README.md

```diff
@@ -209,7 +209,7 @@ linker = "rust-lld"
 
 ## Minimum Supported Rust Version (MSRV)
 
-This crate is guaranteed to compile on stable Rust 1.60.0 and up. It *might*
+This crate is guaranteed to compile on stable Rust 1.64.0 and up. It *might*
 compile with older versions but that may change in any new patch release.
```

Comment:
this `*might*` is now irrelevant (with `rust-version`), we could consider removing this sentence